### PR TITLE
Full Update de_de.json

### PR DESCRIPTION
**Pull request by Creepingpasta**

### CODE REVIEW

File: src/main/resources/assets/twilightforest/lang/de_de.json [hunk 1]
```diff
@@ -1,139 +1,201 @@
...
+  "advancement.twilightforest.magic_map_focus.desc": "Stelle den magischen Kartenfokus mit einer Rabenfeder, Glowstonestaub und Fackelbeeren her.",
+
   "advancement.twilightforest.magic_map": "Bis zum Horizont",
   "advancement.twilightforest.magic_map.desc": "Stelle eine magische Karte her.",
   "advancement.twilightforest.maze_map": "Nichts wie raus hier",
   "advancement.twilightforest.maze_map.desc": "Stelle die Labyrinthkarte her, nachdem du einen Labyrinthkartenfokus gefunden hast.",
   "advancement.twilightforest.ore_map": "Ist es das denn wert?",
   "advancement.twilightforest.ore_map.desc": "Stelle die Labyrinth-Erzkarte her.",
+
+
```
Comment (GizmoTheMoonPig) [inline src/main/resources/assets/twilightforest/lang/de_de.json]: Lots of strange spacing throughout the file, much like this. You dont need this much spacing between blocks of stuff. See the en_us file, all the spacing there is neat and makes sense.

File: src/main/resources/assets/twilightforest/lang/de_de.json [hunk 2]
```diff
@@ -243,201 +326,217 @@
...
-  "block.twilightforest.sort_sign": "Sortingwood Sign",
-  "block.twilightforest.sort_wall_sign": "Sortingwood Wall Sign",*/
-
-  //"block.steeleaf.tooltip": "Reduces fall damage by 25%",
-  //"block.arctic.tooltip": "Reduces fall damage by 90%",
-  //"block.knightmetal.tooltip": "Works as a stronger cactus",
+  "block.twilightforest.quest_ram_trophy": "Schafbock-der-Schatzsuche-Trophäe",
+
+  "block.twilightforest.keepsake_casket": "Andenkenschatulle",
+
```
Comment (GizmoTheMoonPig) [inline src/main/resources/assets/twilightforest/lang/de_de.json:385]: missing the 2 casket messages here

File: src/main/resources/assets/twilightforest/lang/de_de.json [hunk 3]
```diff
@@ -243,201 +326,217 @@
...
+  "structure.hollow_hill": "Hohlhügel",
+  "structure.quest_grove": "Schatzsuche-Hain",
+  "structure.mushroom_tower": "Pilz-Burg",
+  "structure.naga_courty": "Naga-Hof",
+  "structure.lich_tower": "Lich-Turm",
+  "structure.minotaur_labyrinth": "Minotaurus-Labyrinth",
+  "structure.hydra_lair": "Hydra's Lair",
+  "structure.goblin_stronghold": "Goblin-Hochburg",
+  "structure.dark_tower": "Dunkler Turm",
+  "structure.yeti_cave": "Yeti-Höhle": "Yeti-Höhle",
```
Comment (GizmoTheMoonPig) [inline src/main/resources/assets/twilightforest/lang/de_de.json]: this line is improperly formatted, please fix this

File: src/main/resources/assets/twilightforest/lang/de_de.json [hunk 4]
```diff
@@ -606,45 +707,45 @@
...
-  //"item.twilightforest.arctic_armor.tooltip": "Dyeable like Leather Armor",
+  "item.twilightforest.arctic_armor.tooltip": "Einfärbbar wie Lederrüstungen",
 
-  "itemGroup.twilightforest": "Twilight-Forest",
 
+  // Entitäten
   "entity.twilightforest.wild_boar": "Wildschwein",
   "entity.twilightforest.bighorn_sheep": "Dickhornschaf",
-  "entity.twilightforest.deer": "Wilder Hirsch",
+  "entity.twilightforest.deer": "Hirsch",
```
Comment (GizmoTheMoonPig) [inline src/main/resources/assets/twilightforest/lang/de_de.json:716]: The deer is called "Wild Deer", so im not sure if this change was intended or not

**Review by GizmoTheMoonPig — CHANGES_REQUESTED** [review]
Looks good overall, just please address the issues I pointed out in my comments and we'll be good to go! Also just curious, why did you open a new PR instead of just editing the one you previously opened?

**Comment by Creepingpasta** [comment]
> Looks good overall, just please address the issues I pointed out in my comments and we'll be good to go! Also just curious, why did you open a new PR instead of just editing the one you previously opened?

Much tougher than I thought but I just checked every detail now it starts with line 1 and ends with line 1315 just like en_us. Besides the ones you found, I found and fixed some other issues as well.
Hmm, I wasn't looking and thought it was closed. I hope it didn't bring you much trouble.

**Comment by GizmoTheMoonPig** [comment]
much better, thank you. I also noticed you found a bunch of weird translation key errors too... not sure how I didnt see those, but thanks for fixing. Everything looks good now, so I think we're all set to merge! Thanks for the help!

**Pull request merged**